lang/gcc10-bootstrap/Portfile

```diff
@@ -200,10 +209,6 @@ merger_arch_flag            yes
 merger_arch_compiler        yes
 merger_must_run_binaries    yes
 
-if {![info exists universal_possible]} {
-    set universal_possible [expr {${os.universal_supported} && [llength ${configure.universal_archs}] >= 2}]
```

Comment:
You had removed the onle `set universal_possible` which was in this portfile. This means that condition `if {${universal_possible} ...` is always false.

Reply:
This compact code is already part of Macports-base

Reply:
Got it, thanks